services/hanke-service/build.gradle.kts

```diff
@@ -120,6 +122,8 @@ dependencies {
 	annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
 }
 
+jacoco { toolVersion = "0.8.7" }
```

Comment:
The latest release is 0.8.10 (https://github.com/jacoco/jacoco/releases)

Reply:
Updated.